core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/NativeImageResourceBuildItem.java

```diff
@@ -21,6 +24,23 @@ public NativeImageResourceBuildItem(List<String> resources) {
         this.resources = new ArrayList<>(resources);
     }
 
+    public void addResourcePatterns(String identifier, String... resourcePatterns) {
+        this.resourcePatterns = Arrays.asList(resourcePatterns);
+        this.identifier = identifier;
```

Comment:
Hello @RotBolt, is this `addResourcePatterns` allowed to be called multiple times on the same `NativeImageResourceBuildItem` instance? If so, won't that end up overwriting the previous resource patterns and the identifier? If this is intentionally supposed to be resetting/overwriting the previous values, then perhaps this method should be named something like `setResourcePatterns`?

Reply:
This will overwrite the previously set resource patterns.
I will change the Name